src/Snippets/Docs/Testing.cs

```diff
@@ -58,4 +62,117 @@ public static void GetPipelineDescriptorGeneric()
 
         #endregion
     }
+
+    /// <summary>
+    /// A dummy interface to represent a downstream call.
+    /// </summary>
+    private interface IDownstream
+    {
+        /// <summary>
+        /// The method.
+        /// </summary>
+        /// <param name="token">The parameter.</param>
+        /// <returns>The result.</returns>
+        ValueTask<HttpResponseMessage> CallDownstreamAsync(CancellationToken token);
+    }
+
+    private class SUT
```

Comment:
Not everyone is familiar with _SUT_ for _System Under Test_ - can we rename this to something else?